src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -27,10 +27,12 @@
 #include "inference_profiler.h"
 
 #include <math.h>
+
```

Comment:
We still need to update documentation somewhere. Looks like at a minimum we need to change line 482 of perf_analyzer.cc

Reply:
updated